tests/migration/online_migration/post_migration.pm

```diff
@@ -36,6 +37,12 @@ sub run {
     diag "SUSEConnect --status-text: $out";
     assert_script_run "SUSEConnect --status-text | grep -v 'Not Registered'" unless get_var('MEDIA_UPGRADE');
 
+    # enable multiversion for kernel-default based on bsc#1097111, for migration continuous cases only
+    if (get_var('FLAVOR', '') =~ /Continuous-Migration/) {
```

Comment:
Then the cmd can be updated to 
disable_kernel_multiversion_temporarily if (get_var('FLAVOR', '') =~ /Continuous-Migration/);